qemu/tests/cfg/blockdev_commit_server_down.cfg

```diff
@@ -13,23 +13,23 @@
 
     # The following is specified for a local data image 'data',
```

Comment:
@aliang123 Please also update the comments. Thanks

Reply:
Have updated, thanks.